src/main/shacl/inferApplicableUnits.ttl

```diff
@@ -65,8 +65,8 @@ sq:qkRule
                             GROUP BY $this ?qk
                         }
                         FILTER EXISTS {
-                            ?unit qudt:hasQuantityKind ?qk
-                            ?unit a qudt:Unit
+                            ?unit qudt:hasQuantityKind ?qk .
+                            ?unit rdf:type qudt:Unit .
```

Comment:
The same statement (`?unit rdf:type qudt:Unit`) is also needed in line 79. I did that in a later commit, so don't worry about it.